### PR TITLE
RD-5330 Finalize TS migration of Cypress system tests

### PR DESCRIPTION
**Pull request by qooban**
## Description
I've noticed that we are close to finalize TS migration of Cypress integration specs, so created to RD-5330 to finalize it. In total migrated 10 files in this PR.

As `cluster_status` related specs required more typing changes outside of the integration tests, I have created another PR for it - #2045 which extends this one.

## Screenshots / Videos
N/A

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3083/) - Getting Started spec failed, so I'm considering it as PASSED (as I didn't change this spec as well as I didn't change any Cypress support commands)

## Documentation
N/A

### CODE REVIEW

**Review by NorbertAmartus — APPROVED** [review]
Nice piece of work! 💪 
Day by day we're getting closer to have a fully migrated repo 🎉

**Comment by qooban** [comment]
@NorbertAmartus Thanks for approval. It seems I need to "polish" this PR a bit as `pages_edit_spec.ts` with my fix is not always passing. I'll continue next week.

File: test/cypress/integration/templateManagement/pages_edit_spec.ts [hunk 1]
```diff
@@ -47,18 +48,21 @@ describe('Page management', () => {
...
+        cy.get('.modal input[type=text]').type('2');
         cy.get('.modal .toggle').click();
         cy.get('.modal').contains('Save').click();
 
         cy.log('Saving page');
+        cy.intercept('GET', '/console/templates/pages').as('fetchPages');
         cy.contains('Save').click();
 
         cy.log('Veryfiying page row');
+        cy.wait('@fetchPages');
```
Comment (qooban) [inline test/cypress/integration/templateManagement/pages_edit_spec.ts:60]: Waiting for pages fetching (defined in L56) makes it more robust solution as during fetching we are sure that the loader is turned on => L61 works properly.

**Review by NorbertAmartus — APPROVED** [review]
LGTM 🎸

**Pull request merged**